### PR TITLE
[11.0][MIG] account_invoice_blocking

### PR DESCRIPTION
**Pull request by astirpe**
Porting of module account_invoice_blocking to V11

### CODE REVIEW

**Comment by astirpe** [comment]
@pedrobaeza can I ask you the label "Needs review" for this PR?

**Comment by pedrobaeza** [comment]
Done

**Review by melroy89 — APPROVED** [review]

File: account_invoice_blocking/models/account_invoice.py [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+        """
+        This method searches for payable or receivable move line
+        of the invoice
+        :returns payable or receivable move line of the invoice
+        """
+        self.ensure_one()
+        type_receivable = self.env.ref('account.data_account_type_receivable')
+        type_payable = self.env.ref('account.data_account_type_payable')
+        user_type_id_list = [type_receivable.id, type_payable.id]
+        return self.env['account.move.line'].search([
```
Comment (feketemihai) [inline account_invoice_blocking/models/account_invoice.py:22]: @astirpe Can we check also that the move account is equal to the invoice account, we have advance invoices where we also have a payable/receivable on the invoice line, so i want to trace that one

**Review by feketemihai — CHANGES_REQUESTED** [review]

File: account_invoice_blocking/models/account_invoice.py (hunk 1 above)
Comment (astirpe) [reply account_invoice_blocking/models/account_invoice.py:22]: @feketemihai I'm quite busy this period, I'm afraid I will not able to look at it during the next weeks. If you already have implemented a working solution, please open a PR to this branch and I will merge it asap. Thank you!

**Review by yung-wang — APPROVED** [review]
Works fine for the vendor invoices.

**Comment by astirpe** [comment]
Thanks to you all for the reviews!
My initial intention with this PR was the pure porting of the module. There are some requests for extra functionalities that I don't have time to develop at this moment.
Is it OK to merge this porting as it is, and later on open other PRs for the extra improvements?
I would like to close this old PR, thanks.

**Review by tafaRU — APPROVED** [review]
LGTM (only technical review)

**Comment by tafaRU** [comment]
@feketemihai can we merge it?

**Comment by astirpe** [comment]
I would say it's time to merge anyway ...

**Comment by tafaRU** [comment]
Merging as no answer.

**Comment by astirpe** [comment]
Thanks!

**Pull request merged**